### DISCUSSION TITLE
[Feature] Storage template support {{#if album}} ... {{/if}}

### DISCUSSION THREAD

**Discussion by feyst** [immich-app/immich#11999] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Some time ago a feature was added to add {{album}} to the storage template path, this allows for a very nice folder structure. In my case I have 2024/Album-name as path. But not every image is inside an album. This means that my folder now looks like this:
```
...
└── 2024
    ├── 20240801_175952.jpg
    ├── Album-name 
...
```
The 2024 directory now has a mix of folders and images. I would love to be able to do something like this:
```
{{y}}/{{#if album}}{{album}}{{else}}other/{{mm}}{{/if}}
```
Or something similar. This way I have more control over what happens with images that do not belong to an album and I don't have to scroll to a large list of images to find my albums.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#6810

**Reply by feyst** [reply]
It is related but a completely different topic, please don't just close my request. This could be a good intermediate step towards better album support.
The topic I'm discussing here is to check **if** an album exists, not **which** album should be the one selected in the file path. Those are 2 different topics although related not the same.
I've created a PR as well: https://github.com/immich-app/immich/pull/12000
